apps/expo/src/utils/auth-hooks.tsx

```diff
@@ -0,0 +1,48 @@
+import * as Linking from "expo-linking";
+import * as Browser from "expo-web-browser";
+
+import { api } from "./api";
+import { deleteToken, setToken } from "./session-store";
+
+export const signIn = async () => {
+  const signInUrl = "http://localhost:3000/api/auth/signin";
+  const redirectTo = "exp://192.168.10.181:8081/login";
```

Comment:
gotta figure these URLs out

Reply:
The exp://192 is your local computer address! 
You can use manifest.debuggerHost for that

Reply:
in dev yes, but what about when you don't have a debuggerhost